Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/Dockerfile-trainer

```
@@ -9,8 +9,6 @@ ENV GCP_CREDS_JSON_BASE64_ENV=${GCP_CREDS_JSON_BASE64}
 ENV GOOGLE_APPLICATION_CREDENTIALS=/root/gcp.json
 RUN echo "${GCP_CREDS_JSON_BASE64_ENV}"| base64 --decode > "${GOOGLE_APPLICATION_CREDENTIALS}"
 
-COPY requirements-torchserve.txt requirements.txt /tmp/
+COPY requirements.txt /tmp/
 RUN pip install --upgrade pip && \
-    pip install --no-cache-dir torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio==0.7.2 torchserve torch-model-archiver -f https://download.pytorch.org/whl/torch_stable.html && \
-    pip install --no-cache-dir -r /tmp/requirements.txt \
-    pip install --no-cache-dir -r /tmp/requirements-torchserve.txt
+    pip install --no-cache-dir -r /tmp/requirements.txt
```

Same as Dockerfile-torchserve: maybe we should use official repo.
```suggestion
    pip install --no-cache-dir -r /tmp/requirements.txt -f https://download.pytorch.org/whl/torch_stable.html
```

---

Will be resolved by base image using serve install_dependencies